.env.dev

```diff
@@ -1 +1,16 @@
+# Docker compose
+COMPOSE_FILE=docker-compose.local.yml
+
+# web
 DEBUG=1
+
+# graphql-engine
+HASURA_GRAPHQL_ADMIN_SECRET=dev_only_password
+HASURA_GRAPHQL_DATABASE_URL=postgres://zagrajmy:zagrajmy@postgres:5432/zagrajmy_dev
```

Comment:
If we moved postgres conf above, is it possible to to use $(POSTGRES_DB) etc here?

Reply:
ok

Reply:
@lprasol 
Looks like inside `.env` we cannot reference other variables.